schema.json

```diff
@@ -1338,7 +1338,7 @@
         {
           "kind": "SCALAR",
           "name": "DateTime",
-          "description": "An ISO-8601 encoded UTC date time string.",
+          "description": "An ISO-8601 encoded UTC date time string. Example value: `\"2019-07-03T20:47:55Z\"`.",
```

Comment:
Maybe we could try to have current version as the example?

Reply:
This is pulling from Storefront API documentation. You are welcome to change the description there, if you'd like.